lib/signalman.rb

```diff
@@ -90,7 +90,8 @@ def skip?
   class QueryHandler < BaseHandler
     def skip?
       ["SCHEMA", "TRANSACTION"].include?(event.payload[:name]) ||
-        event.payload[:name]&.include?("Signalman::")
+        event.payload[:name]&.include?("Signalman::") ||
+        !ActiveRecord::Base.connection.table_exists?(:signalman_events)
```

Comment:
```suggestion
        self.class.events_table?
        
        def self.events_table?
            return @events_table if defined?(@events_table)
          @events_table = !ActiveRecord::Base.connection.table_exists?(:signalman_events)
        end
```
How about just doing the lookup once and memoizing the result?